### PR TITLE
US018 Create a Post Operation to Create a New User

### PR DESCRIPTION
**Pull request by RobertoPloneda**

### CODE REVIEW

File: src/test/java/com/at/globalclasses/MongoDBConnection.java [hunk 1]
```diff
@@ -87,12 +87,16 @@ public boolean compareID(String collection, String id) {
 
     public String getObjectByID(String collection, String id) {
         String mObject = "";
+        JSONObject json = new JSONObject(id);
+        String idValue = json.getString("id");
         MongoCollection<Document> coll = mDataBase.getCollection(collection);
-        FindIterable<Document> findIterable = coll.find(Filters.eq("_id", new ObjectId(id)));
+        ObjectId objectId = new ObjectId(idValue);
+        FindIterable<Document> findIterable = coll.find(Filters.eq("_id", objectId));
```
Comment (ramiroAT) [inline src/test/java/com/at/globalclasses/MongoDBConnection.java]: You could have left the method just as it was. These lines that you added are not needed.

File: src/test/java/com/at/globalclasses/MongoDBUtils.java [hunk 2]
```diff
@@ -22,6 +23,16 @@ public static String getRandomID(String env, String mDataBase, String collection
         return id;
     }
 
+
+    public static String compareID(String env, String mDataBase, String collection, String response) {
+        MongoDBConnection db = new MongoDBConnection(env, mDataBase);
+
```
Comment (ramiroAT) [inline src/test/java/com/at/globalclasses/MongoDBUtils.java]: Here you can extract the id string value before to call getObjectById. Also this method is not compating anything at all so could be renamed.

**Review by ramiroAT — COMMENTED** [review]
Did you also work in POst method?

File: src/test/java/com/at/globalclasses/MongoDBConnection.java (hunk 1 above)
Comment (RobertoPloneda) [reply src/test/java/com/at/globalclasses/MongoDBConnection.java]: I put it the way it was before in the new commit

File: src/test/java/com/at/globalclasses/MongoDBUtils.java (hunk 2 above)
Comment (RobertoPloneda) [reply src/test/java/com/at/globalclasses/MongoDBUtils.java]: I changed the name and i extracted the id from here and i send the id value to getObjectById

**Pull request closed without merging**